libphx.code-workspace

```diff
@@ -0,0 +1,30 @@
+{
+	"folders": [
+		{
+			"path": "./"
+		}
+	],
+	"settings": {
+		"Lua.workspace.maxPreload": 10000,
+		"Lua.diagnostics.libraryFiles": "Disable",
+		"Lua.runtime.version": "LuaJIT",
+		"Lua.format.enable": true,
+		"Lua.format.defaultConfig":{
+			"indent_style":"space",
+			"indent_size":"4"
+		},
+		"Lua.misc.parameters": [
+			"--preview"
+		],
+		"editor.codeActionsOnSave": {
+			"source.fixAll.eslint": true
+		},
+		"eslint.format.enable": true,
+		"Lua.completion.callSnippet": "Disable"
+	},
+	"extensions": {
+		"recommendations": [
+			"sumneko.lua"
```

Comment:
Add rust-analyzer extension for Rust code.
I also find helpfull following extensions: crates (for Rust), CodeLLDB (for debugging Rust code), Even Better Toml, GitLens, Markdown All In One, TODO Highlight

Reply:
added that rust-analyzer just now :)

can maybe @dgavedissian and you decide on a list / add this to the workspace?

Reply:
Last 4 extensions are generic, so it can be useful for Lua workspace as well

Reply:
u use those too so i have nothing against it :)

Reply:
i recommend todo tree btw, works better for me ;)

Reply:
Yeah, and it was updated recently. Just switched to it. Thanks for recommendation! :+1: